packages/realm/RealmJS.podspec

```diff
@@ -35,36 +37,66 @@ Pod::Spec.new do |s|
 
-  s.source_files           = 'react-native/ios/RealmReact/*.mm'
-  s.public_header_files    = 'react-native/ios/RealmReact/*.h'
+  s.source_files           = 'react-native/ios/RealmReact/*.mm',
+                             'react-native/shared/*.cpp',
+                             'binding/ios/platform.mm'
+
+  s.public_header_files    = 'react-native/ios/RealmReact/*.h',
+                             'react-native/ios/build/include/**/*.{h,hpp}'
 
   s.frameworks             = uses_frameworks ? ['React'] : []
 
   s.library                = 'c++', 'z', 'compression'
 
   s.pod_target_xcconfig    = {
                                 # Setting up clang
-                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++17',
+                                'CLANG_CXX_LANGUAGE_STANDARD' => 'c++20',
                                 'CLANG_CXX_LIBRARY' => 'libc++',
                                 # Setting the current project version and versioning system to get a symbol for analytics
                                 'CURRENT_PROJECT_VERSION' => s.version,
                                 'VERSIONING_SYSTEM' => 'apple-generic',
+                                'GCC_PREPROCESSOR_DEFINITIONS' => '$(inherited) REALM_ENABLE_SYNC=1',
+                                'GCC_SYMBOLS_PRIVATE_EXTERN' => 'YES',
                                 # Header search paths are prefixes to the path specified in #include macros
                                 'HEADER_SEARCH_PATHS' => [
                                   '"$(PODS_TARGET_SRCROOT)/react-native/ios/RealmReact/"',
-                                  '"$(PODS_ROOT)/Headers/Public/React-Core/"'
-                                  #"'#{app_path}/ios/Pods/Headers/Public/React-Core'" # Use this line instead of 👆 while linting
+                                  '"$(PODS_TARGET_SRCROOT)/react-native/ios/build/include/"',
+                                  '"$(PODS_TARGET_SRCROOT)/binding/"',
+                                  '"$(PODS_TARGET_SRCROOT)/bindgen/src/"',
+                                  '"$(PODS_TARGET_SRCROOT)/bindgen/vendor/realm-core/bindgen/src/"'
                                 ].join(' ')
                               }
+  # Create placeholders for vendored_libraries, so they are added to the xcode project
+  s.prepare_command = <<-EOS
+  source "#{__dir__}/scripts/generate-dummy-libs.sh"
+  source "#{__dir__}/scripts/generate-input-list.sh"
+  EOS
 
-  # TODO: Consider providing an option to build with the -dbg binaries instead
-  s.vendored_frameworks = 'react-native/ios/realm-js-ios.xcframework'
+  s.vendored_libraries = "react-native/ios/lib/librealm.a",
+                         "react-native/ios/lib/librealm-object-store.a",
+                         "react-native/ios/lib/librealm-parser.a",
+                         "react-native/ios/lib/librealm-sync.a"
 
   s.dependency 'React'
+
+  CMAKE_PATH = Pod::Executable::which!('cmake')
```

Comment:
Use `witch` instead of `witch!` to avoid throwing if it isn't found: https://rubydoc.info/gems/cocoapods/Pod/Executable#which-class_method